src/EventTeachingService.h

```diff
@@ -7,29 +7,30 @@
 
 #include "Service.h"
 
-namespace VLCB
-{
+namespace VLCB {
 
 class Configuration;
 
-class EventTeachingService : public Service
-{
+class EventTeachingService : public Service {
 public:
   virtual void setController(Controller *cntrl) override;
   virtual Processed handleMessage(unsigned int opc, CANFrame *msg) override;
 
-  virtual byte getServiceID() override { return 4; }
-  virtual byte getServiceVersionID() override { return 1; }
-  
+  virtual byte getServiceID() override {
+    return 4;
+  }
+  virtual byte getServiceVersionID() override {
+    return 1;
+  }
```

Comment:
Prefer to keep these one-liners on a single line when they are in a header file.

Reply:
Autoformat, again!